src/OrchardCore.Modules/OrchardCore.ContentLocalization/Startup.cs

```diff
@@ -47,6 +57,16 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddScoped<IPermissionProvider, Permissions>();
             services.AddScoped<IAuthorizationHandler, LocalizeContentAuthorizationHandler>();
         }
+
+        public override void Configure(IApplicationBuilder app, IEndpointRouteBuilder routes, IServiceProvider serviceProvider)
```

Comment:
I already done this in another PR, could you rebase on dev?

Reply:
This was my intention ;)